src/ngSanitize/sanitize.js

```diff
@@ -158,7 +158,7 @@ function $SanitizeProvider() {
     return function(html) {
       var buf = [];
       htmlParser(html, htmlSanitizeWriter(buf, function(uri, isImage) {
```

Comment:
In that case I can handle and document the google3-specific issues, since that's likely to be plumbed differently internally. There's still going to be a few issues for everyone:

- if we keep this common whitelist model, all URL bindings of blob urls that were previously fine will break without $sce.trustAsUrl, and I don't think we can figure these out beforehand.
- whitelist customizations will break (renamed)
- things plugging directly into Angular's $set for URLs will lose sanitization (in line with other contexts).